ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml

```diff
@@ -568,6 +568,13 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-external-aws
+- as: k8s-e2e-aws-hpa
```

Comment:
This could be done as a follow-up PR, but I suggest you don't use the `k8s-` prefix because that implies the job is using ginkgo.